Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -1186,7 +1187,10 @@ def get_width_height(args: dict):
 
 
 def main():  # pragma: no cover
-    demisto.debug(f"main, {demisto.command()=}")
+    incident = demisto.incident()
+    incident_id = incident.get('id')
+    incident_name = incident.get('name')
+    demisto.debug(f"main, {demisto.command()=}, {incident_id=}, {incident_name=}")
```

Comment:
We might not be in an incident (e.g. the general War Room)
```suggestion
    incident_id = ''
    incident_name = ''
    incident = demisto.incident()
    incident_id = incident.get('id')
    if incident_id:
        incident_name = incident.get('name')
    demisto.debug(f"main, {demisto.command()=}, {incident_id=}, {incident_name=}")
```